randovania/games/super_metroid/logic_database/Maridia.txt

```diff
@@ -607,47 +607,78 @@ Extra - asset_id: 145
+              # Get up
+              All of the following:
+                  Gravity Suit
+                  Any of the following:
+                      Space Jump or Snail Climb (Beginner) or Wall Jumps (Beginner)
+                      # Spring Ball Jump
+                      Hi-Jump Boots and Spring Ball Jump (Intermediate) and Can Use Spring Ball
+                      # IBJ
+                      Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
   > Door to Botwoon Hallway
-      All of the following:
-          Gravity Suit
-          Grapple Beam or Space Jump
+      Any of the following:
+          All of the following:
+              Gravity Suit
+              Any of the following:
+                  Grapple Beam or Space Jump or Wall Jumps (Advanced)
+                  # Shinespark Up
+                  Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 130
+                  # IBJ
+                  Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
+          # Snail Climb
+          Snail Climb (Expert)
   > Door to Below Botwoon Energy Tank
-      All of the following:
-          Gravity Suit
-          Grapple Beam or Space Jump
+      Any of the following:
+          All of the following:
+              Gravity Suit
+              Any of the following:
+                  Grapple Beam or Space Jump or Wall Jumps (Intermediate)
+                  All of the following:
+                      Speed Booster
+                      Any of the following:
+                          # HJ up or Shinespark
+                          Hi-Jump Boots
+                          Shinespark (Beginner) and Shinespark Damage ≥ 50
+                  # IBJ
+                  Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
+          # Snail Climb
+          Snail Climb (Intermediate)
   > Door to East Aqueduct Quicksand Room
       Trivial
   > Pickup (Missile Expansion)
-      Gravity Suit and Speed Booster
+      Any of the following:
+          # Shinespark
+          Gravity Suit and Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 100
+          # Snail Climb and Clip: https://youtu.be/ud0I-bb6mVw
+          Movement (Expert) and Snail Climb (Intermediate)
 
 > Door to Botwoon Hallway; Heals? False
   * Layers: default
   * Normal Door to Botwoon Hallway/Door to Aqueduct
   > Door to Crab Shaft
-      All of the following:
-          Gravity Suit
-          Any of the following:
-              Screw Attack
-              All of the following:
-                  Morph Ball
-                  Morph Ball Bombs or Power Bombs
+      Can Lay Bombs
   > Door to Aqueduct Save Room
       Trivial
 
 > Door to Below Botwoon Energy Tank; Heals? False
   * Layers: default
   * Normal Door to Below Botwoon Energy Tank/Door to Aqueduct
   > Door to Aqueduct Save Room
-      Gravity Suit
+      Any of the following:
+          Gravity Suit or Snail Climb (Beginner)
+          # Downgrab
```

Comment:
TODO: video ?

Reply:
For the downgrab over this? 
![image](https://github.com/randovania/randovania/assets/46821362/084845ea-aaa6-4bbf-8049-95a2850dc2b6)

Reply:
ye